packages/tools/devtools/devtools-core/src/ContainerDevtools.ts

```diff
@@ -447,7 +459,10 @@ export class ContainerDevtools implements IContainerDevtools, HasContainerKey {
 		this.dataVisualizer =
 			props.containerData === undefined
 				? undefined
-				: new DataVisualizerGraph(props.containerData, defaultVisualizers);
+				: new DataVisualizerGraph(props.containerData, {
+						...defaultVisualizers,
+						...props.dataVisualizers, // User-specified visualizers take precedence over system defaults
```

Comment:
This was removed in a recent PR of mine. The input `props` should no longer have `dataVisualizers`.

Reply:
I tried doing some of the merging but I think I messed something up. Would like to look at it during our data editing sync